lib/api/redfish-1.0/event-service.js

```diff
@@ -0,0 +1,266 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+var urlParse = require('url-parse');
+var injector = require('../../../index.js').injector;
+var redfish = injector.get('Http.Api.Services.Redfish');
+var schemaApi = injector.get('Http.Api.Services.Schema');
+var Promise = injector.get('Promise'); // jshint ignore:line
+var _ = injector.get('_');  // jshint ignore:line
+var Constants = injector.get('Constants');
+var RedfishTool = injector.get('JobUtils.RedfishTool');
+var controller = injector.get('Http.Services.Swagger').controller;
+var Errors = injector.get('Errors');
+var messenger = injector.get('Services.Messenger');
+var Logger = injector.get('Logger');
+var uuid = injector.get('uuid');
+var timeoutInSeconds = 10;
+var retryAttempts = 3;
+var eventSubscriptions = {};
+var subscription;
+var logger = Logger.initialize('event-service');
+
+var eventListener = function(callback) {
```

Comment:
You could also just represent this as `function eventListener` rather than `var eventListener = function`. Same for other functions below. If you do it that way, then the location of the definition in the file doesn't matter for having access to the function name in this scope.